### ISSUE TITLE
[FEATURE] API to disable selection

### ISSUE THREAD

**Issue by DylanXie123** [arnog/mathlive#1136]
## Description

I want to disable selection in math-field, but css property `userSelect: 'none'` doesn't seems to work with mathlive, I guess mathlive has its own system to handle selection? And I look through the documentation without finding any API to disable selection, I wonder if it's possible to add an API to disable selection.

## Steps to Reproduce

React Code Example
```js
import MathView from 'react-math-view'; // just a wrapper to use mathlive in react

function MathKey(content: string) {
  return (
    <MathView
      value={content}
      readOnly={true}
      style={{ userSelect: 'none' }}
    />
  );
}
```

### Actual Behavior

Can still select content when set `userSelect: 'none'` .
![untitled](https://user-images.githubusercontent.com/29018622/129690340-f66a159b-80aa-473c-b948-40b3a804e9c5.png)

### Expected Behavior

An new API to disable selection.

### Environment

**MathLive version**
0.69.4

**Operating System** 
Windows10

**Browser** 
Chrome

**Comment by arnog** [comment]
Alternatively, the math-field component should respect the `user-select: none` property, since that seems the reasonable and intuitive thing to do.

**Comment by DylanXie123** [comment]
Yes, I agree that `user-select: none` is a better way to do this.

**Comment by arnog** [comment]
Fixed in mathlive 0.90.0
